addons/sale_timesheet/views/sale_timesheet_portal_templates.xml

```diff
@@ -58,9 +61,15 @@
         <th name="t_label" position="before">
             <th t-if="not groupby == 'so_line'">Sales Order Item</th>
         </th>
+        <th name="t_label" position="before">
+            <th t-if="not groupby == 'timesheet_invoice_id'">Invoice</th>
+        </th>
         <xpath expr="//tbody//td[hasclass('text-end')]" position="before">
             <td t-if="not groupby == 'so_line'"><span t-field="timesheet.so_line" t-att-title="timesheet.so_line.display_name"></span></td>
         </xpath>
+        <xpath expr="//tbody//td[hasclass('text-end')]" position="before">
+            <td t-if="not groupby == 'timesheet_invoice_id'"><span t-field="timesheet.timesheet_invoice_id" t-att-title="timesheet.timesheet_invoice_id.display_name"></span></td>
+        </xpath>
```

Comment:
Why not doing that? 
```suggestion
            <td t-if="not groupby == 'timesheet_invoice_id'"><span t-field="timesheet.timesheet_invoice_id" t-att-title="timesheet.timesheet_invoice_id.display_name"></span></td>
        </xpath>
```

Reply:
@xavierbol yes, we can :100: 
Thanks for looking into it :+1: :slightly_smiling_face: